src/osvr/PluginHost/SearchPath.cpp

```diff
@@ -172,11 +172,20 @@ namespace pluginhost {
                 }
                 const auto pluginBaseName =
                     pluginCandidate.filename().stem().generic_string();
+
+#if defined(_MSC_VER) && !defined(NDEBUG)
+                // Visual C++ debug runtime: we append to the plugin name.
+                const std::string decoratedPluginName =
+                    pluginName + OSVR_PLUGIN_DEBUG_SUFFIX;
+#else
+                const std::string &decoratedPluginName = pluginName;
```

Comment:
Why is `decoratedPluginName` a reference here but not on line 178 above? (Actually, I know the answer. But it's probably better to store a copy in both cases for consistency's sake.)

Reply:
That's probably because it's a copypasta from its original location. I moved this piece of code from RegistrationContext.cpp. Good catch! I'll change that